Andlet/Andlet/Views/Offers/OfferView.swift

```diff
@@ -48,6 +48,12 @@ struct OfferView: View {
                     Text("\(1) 🛁")
                         .font(.custom("LeagueSpartan-SemiBold", size: 16))
                         .foregroundColor(Color(hex: "000000"))
+                    Text("|")
+                        .font(.custom("LeagueSpartan-SemiBold", size: 16))
+                            .foregroundColor(Color(hex: "000000"))
+                    Text("\(3) 🧑‍🤝‍🧑")
```

Comment:
Does this Icon will not provoke any crashes in the app?

Reply:
No!! This icon is used to include the smart feature in the app